h/util/metrics.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import unicode_literals
+import newrelic.agent
```

Comment:
```suggestion

import newrelic.agent
```

This should be caught by our linter (will have to investigate some time why it isn't) but there should be a blank line between standard library imports (`from __future__ import unicode_literals`) and third-party imports (`import newrelic.agent`). See https://www.python.org/dev/peps/pep-0008/#imports